Embedded Metadata.js

```diff
@@ -766,25 +766,27 @@ function getAuthorFromByline(doc, newItem) {
 
 function finalDataCleanup(doc, newItem) {
 	/**If we already have tags - run through them one by one,
-	 * split where ncessary and concat them.
-	 * This  will deal with multiple tags, some of them comma delimited,
+	 * split where necessary and concat them.
+	 * This will deal with multiple tags, some of them comma delimited,
 	 * some semicolon, some individual
 	 */
-	if (newItem.tags.length && Zotero.parentTranslator) {
-		var tags = [];
-		for (var i in newItem.tags) {
-			newItem.tags[i] = newItem.tags[i].trim();
-			if (newItem.tags[i].indexOf(';') == -1) {
-				//split by comma, since there are no semicolons
-				tags = tags.concat( newItem.tags[i].split(/\s*,\s*/) );
-			} else {
-				tags = tags.concat( newItem.tags[i].split(/\s*;\s*/) );
+	if (newItem.tags && newItem.tags.length && Zotero.parentTranslator) {
```

Comment:
The other thing I was thinking is that we could make the comma split dependent on whether `tags.length>1` with the thinking that if there are multiple tags, the comma is likely not a delimiter. Advantage of that would be that it'd work for all EM sites, not just with the option. Downside, obviously, is less flexibility for parent translatoros.

We could also do both that _and_ the preference, although I think that's too convoluted. What are your thoughts between the two?